hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/keys/GetKeyHandler.java

```diff
@@ -90,7 +90,7 @@ protected void execute(OzoneClient client, OzoneAddress address)
 
     if (isVerbose() && !"/dev/null".equals(dataFile.getAbsolutePath())) {
       try (InputStream stream = new FileInputStream(dataFile)) {
-        String hash = DigestUtils.md5Hex(stream);
+        String hash = DigestUtils.sha256Hex(stream);
         out().printf("Downloaded file hash : %s%n", hash);
```

Comment:
Please include the hash type in terminal output, for example:

```
        out().printf("Downloaded file hash (sha256): %s%n", hash);
        out().printf("Downloaded file sha256 checksum: %s%n", hash);
```

Same in PutKeyHandler.

Reply:
@GAOJHIHCYUAN I can merge this in once you address @kaijchen 's review comments

Reply:
I had another PR in #4106
including the hash type in terminal output.